code/modules/spells/spell_types/hivemind.dm

```diff
@@ -284,6 +284,14 @@
 
 	QDEL_NULL(backseat)
 
+	if(original_body && original_body.mind)
```

Comment:
I want to start shortening these but forgot the format

it's either

```DM
original_body?.mind 
```
OR
```DM
original_body?original_body.mind 
```

mega brain toot atm